Sources/Views/InputTextView.swift

```diff
@@ -24,10 +24,32 @@
 
 import UIKit
 
+/**
+ A UITextView that has a UILabel embedded for placeholder text
+ 
+ ## Important Notes ##
+ 1. Changing the font, textAlignment or textContainerInset automatically performs the same modifications to the placeholderLabel
+ 2. Intended to be used in an `MessageInputBar`
+ 3. Default placeholder text is "New Message"
+ 4. Will pass a pasted image it's `MessageInputBar`'s `InputManager`s
+ */
 open class InputTextView: UITextView {
-
+    
     // MARK: - Properties
-
+    
+    open override var text: String! {
+        didSet {
+            textViewTextDidChange()
+        }
+    }
+    
+    open override var attributedText: NSAttributedString! {
+        didSet {
+            textViewTextDidChange()
```

Comment:
Very interesting. I'm sorta against manually calling anything that is normally triggered by the notification center, but let's go for it for now. Reason being is because this kind of results in unexpected behavior

Reply:
If someone manually sets the attributedText property to empty, the placeholder label is still hidden without this

Reply:
Sorry I'm not sure I follow, I still think the `textViewTextDidChange()` body would be better as a `didSet` here 🤔